src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
```

Comment:
From what i understand, all Jitstress modes are controlled by a single seed value (complus_jistress=value).  It would be a natural use of this seed for random in-liner under stress mode = STRESS_INLINE_RANDOM.  Like all other stress areas, GetPolicy() could call compStressCompile(stressArea, weight) and based on it can choose Random in-lining policy for jit compiling a method.